geoips/commandline/ancillary_info/cmd_instructions.yaml

```diff
@@ -214,8 +214,8 @@ instructions:
       List off every GeoIPS Interface. This is useful for seeing what is
       available throughout GeoIPS, but can also be extended to see what interfaces have
       been implemented in other geoips package, or just a certain package. This
-      information can then be used with `geoips get-interface <interface_name>` to get
-      more information about a certain interface.
+      information can then be used with `geoips describe-interface <interface_name>` to
```

Comment:
Should "describe-interface" be "describe interface"?

Reply:
Fixed! Should just be ``geoips describe <interface_name>``. This was outdated apparently.